### PR TITLE
Add a note about “--platform” flag deprecation

### PR DESCRIPTION
**Pull request by amyblais**
Will adjust the text in https://about.mattermost.com/deprecated-features/ once this PR is reviewed

### CODE REVIEW

**Review by isacikgoz — CHANGES_REQUESTED** [review]

**Review by isacikgoz — APPROVED** [review]
LGTM

**Review by cwarnermm — COMMENTED** [review]
Suggested update provided inline.

**Review by cwarnermm — APPROVED** [review]
Looks great!

File: source/administration/important-upgrade-notes.rst [hunk 1]
```diff
@@ -8,7 +8,10 @@ Important Upgrade Notes
    Support for Mattermost Server v5.19 `Extended Support Release <https://docs.mattermost.com/administration/extended-support-release.html>`_ has come to the end of its lifecycle. Upgrading to v5.25 or later is required.
 
 .. important::
-   TLS versions 1.0 and 1.1 have been deprecated by browser vendors. In an upcoming release (version TBD), mmctl will return an error when connected to Mattermost servers deployed with these TLS versions and System Admins will need to explicitly add a flag in their commands to continue to use them. We recommend upgrading to TLS version 1.2 or higher.
+   TLS versions 1.0 and 1.1 have been deprecated by browser vendors. In Mattermost Server v5.32 (February 16), mmctl will return an error when connected to Mattermost servers deployed with these TLS versions and System Admins will need to explicitly add a flag in their commands to continue to use them. We recommend upgrading to TLS version 1.2 or higher.
```
Comment (amyblais) [inline source/administration/important-upgrade-notes.rst:11]: Also added a date for the TLS deprecation

File: source/administration/important-upgrade-notes.rst [hunk 2]
```diff
@@ -8,7 +8,10 @@ Important Upgrade Notes
...
+
+.. important::
+   Customers using the ``mattermost`` binary with the “--platform” flag, or using the ``platform`` binary directly, to run the Mattermost server application via a systemd file or custom script will be required to update to use only the ``mattermost`` binary as the “--platform” flag and the ``platform`` binary will be deprecated in a future release.
```
Comment (wiersgallak) [inline source/administration/important-upgrade-notes.rst:14]: Recommended edits: 

The ``platform`` binary and “--platform” flag will be deprecated in a future release. If you are using the “--platform” flag or using the ``platform`` binary directly to run the Mattermost server application via a systemd file or custom script, you will be required to use only the ``mattermost`` binary.

**Pull request merged**